frontend/src/components/indicatorReport/AddEditIndicatorReport.js

```diff
@@ -148,38 +152,26 @@ export default function AddEditIndicatorReport() {
   };
 
   const validate = () => {
-    const error = {};
-    // if (!form.name)
-    //   error.name = 'The field cannot be empty';
-    // if (!form.comment)
-    //   error.comment = 'The field cannot be empty';
-    // if (!form.organization)
-    //   error.organization = 'The field cannot be empty';
-    // if (!form.indicator)
-    //   error.indicator = 'The field cannot be empty';
-    // if (!form.startTime)
-    //   error.startTime = 'The field cannot be empty';
-    // if (!form.endTime)
-    //   error.endTime = 'The field cannot be empty';
-    // if (form.uri && !isValidURL(form.uri))
-    //   error.uri = 'The field cannot be empty'
-    // if (!!form.startTime && !!form.endTime && form.startTime > form.endTime){
-    //   error.startTime = 'The date must be earlier than the end date'
-    //   error.endTime = 'The date must be later than the start date';
-    // }
-    //
-    // if (!form.numericalValue)
-    //   error.numericalValue = 'The field cannot be empty';
-    if (form.numericalValue && isNaN(form.numericalValue))
-      error.numericalValue = 'The field must be a number';
-    // if (!form.unitOfMeasure)
-    //   error.unitOfMeasure = 'The field cannot be empty';
-    // if (!form.dateCreated)
-    //   error.dateCreated = 'The field cannot be empty';
-    setErrors(error);
-    return Object.keys(error).length === 0;
+    const errors = {};
+    validateForm(form, attriConfig, attribute2Compass, errors, ['uri']);
+    setErrors(errors);
+    return Object.keys(errors).length === 0;
   };
 
+  const attribute2Compass = {
+    name: 'cids:hasName',
+    comment: 'cids:hasDescription',
```

Comment:
looks like the internal key for comment is 'cids:hasComment'. The same mistake goes to organization here, so you may want to check again one by one.